.github/workflows/ci-publish.yaml

```diff
@@ -0,0 +1,83 @@
+# This workflow is responsible to build and publish a release build
+# It triggers once a new GitHub Release is published
+name: Publish CI
+
+on:
+  release:
+    types:
+      - released
+  workflow_dispatch:
+
+jobs:
+  read-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.parse.outputs.version }}
+    steps:
```

Comment:
NGL, it happened at least once during my own testing, but noticed soon enough to cancel it.
Regardless, I agree that this is a solid idea to prevent human error :+1: